### DISCUSSION TITLE
Question Regarding Machine Learning Training Assistant

### DISCUSSION THREAD

**Discussion by BryanGsep** [camicroscope/GSOC#36] [Q&A]
Hi caMicroscope mentor,

I look forward to becoming a contributor to caMicroscope this year by participating in the Machine Learning Training Assistance project. I want to listen to the mentor's ideas in detail about this project.

The focus of the project is "improve the user experience of the training workflow" and "find ways to improve performance and time taken for predictions".

After running and learning the background of caMicroscope, the current "workbench" training is able to use manually labeled data on caMicroscope. In this project, is our goal to allow users to use existing models to perform data labeling and annotation?

Thank you so much.

**Comment by shaiktahseen** [comment]
Hi camicroscope community, I'm Shaik Tahseen. I'm a beginner and have knowledge in Html, CSS, JavaScript & Flexboxes, and I'm curious to join GSOC 2023 this year.
I like to contribute Can anyone help me to know how to join Jenkins & how to contribute & get into camicroscope  as a contributor with basic skills?  I really like to learn, contribute, give my best & look for opportunities where I can upskill myself & also fulfill the goals of the organization.

If anyone let me know what would be really helpful.
I see the closing date for GSOC is 4th April 2023

Here are my bootcamp certifications.
HTML & CSS Essentials Bootcamp
https://verify.letsupgrade.in/certificate/LUEHTMLFEB123307

JavaScript Programming Essentials Bootcamp
https://verify.letsupgrade.in/certificate/LUEJSMAR123657

•	Email : shaiktahseen05@gmail.com
•	GitHub :  https://github.com/shaiktahseen
•	LinkedIn :  https://www.linkedin.com/in/shaik-tahseen-%F0%9F%87%AE%F0%9F%87%B3-39074b17a/
•	Twitter: https://twitter.com/shaiktahseen\_05
